requirements/travis.txt

```diff
@@ -4,22 +4,65 @@
 #
 #    make upgrade
 #
-appdirs==1.4.4            # via -r requirements/tox.txt, virtualenv
-certifi==2020.12.5        # via requests
-chardet==4.0.0            # via requests
-coverage==5.4             # via coveralls
-coveralls==3.0.0          # via -r requirements/travis.in
-distlib==0.3.1            # via -r requirements/tox.txt, virtualenv
-docopt==0.6.2             # via coveralls
-filelock==3.0.12          # via -r requirements/tox.txt, tox, virtualenv
-idna==2.8                 # via -c requirements/constraints.txt, requests
-packaging==20.9           # via -r requirements/tox.txt, tox
-pluggy==0.13.1            # via -r requirements/tox.txt, tox
-py==1.10.0                # via -r requirements/tox.txt, tox
-pyparsing==2.4.7          # via -r requirements/tox.txt, packaging
-requests==2.25.1          # via coveralls
-six==1.15.0               # via -r requirements/tox.txt, tox, virtualenv
-toml==0.10.2              # via -r requirements/tox.txt, tox
-tox==3.21.4               # via -r requirements/tox.txt
-urllib3==1.26.3           # via requests
-virtualenv==20.4.2        # via -r requirements/tox.txt, tox
+appdirs==1.4.4
```

Comment:
Summary of changes:

coverage==5.4 to coverage==5.5
coveralls==3.0.0 to coveralls==3.0.1
tox==3.21.4 to tox==3.23.0
urllib3==1.26.3 to urllib3==1.26.4